### DISCUSSION TITLE
Is there a future intent to support compare range with equal sign?

### DISCUSSION THREAD

**Discussion by nathipg** [handsontable/hyperformula#838] [Feature requests]
I'm not sure about what's the right name of this feature, but I have a example.

=SUMPRODUCT((A:A=H2)*(E:E=G3), D:D)

Is there a future intent to support that "A:A=H2" feature in formulas? Right now it gives a parser error

**Comment by wojciechczerniak** [comment]
Sorry, I'm not familiar with this syntax. Do you have any reference to the description or algorithm how does it work?

**Reply by nathipg** [reply]
I have this excel example:

https://exceltable.com/en/excel-functions/function-sumproduct

If you scroll to "SUMPRODUCT WITH THE CONDITION" there's an example of how it works.

Sorry for not to explain in details, I knew this feature recently too.

**Reply by wojciechczerniak** [reply]
Cool. That's a complete use case we can test on. Thank you

**Reply by izulin** [reply]
Not sure about parsing error, since this is and was supported.
However, there were some minor issues with evaluation of such formulas for column ranges, which should be fixed now.
See: https://github.com/handsontable/hyperformula/pull/839

the example test there should give you a hint on how to use it:
```
    const engine = HyperFormula.buildFromArray([
      [1, 1, 3, '=SUMPRODUCT((A:A=1)*(B:B=1), C:C)'],
      [1, 2, 3],
      [3, 1, 3],
      ], {useArrayArithmetic: true}
    )
```

**Comment by wojciechczerniak** [comment]
For anyone reading this thread: Problem was fixed in #839
